scripts/build_debian_base_system.sh

```diff
@@ -20,9 +20,11 @@ generate_version_file()
 
 if [ "$ENABLE_VERSION_CONTROL_DEB" != "y" ]; then
     if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
-        # qemu arm bin executable for cross-building
-        sudo mkdir -p $FILESYSTEM_ROOT/usr/bin
-        sudo cp /usr/bin/qemu*static $FILESYSTEM_ROOT/usr/bin || true
+        if [ $MULTIARCH_QEMU_ENVIRON == y ]; then
```

Comment:
can you remove line 22?

Reply:
Reference to https://www.debian.org/mirror/list,  debian-archive.trafficmanager.net doesn't have the arm source mirror.  Is the http://deb.debian.org/debian used for  the source  for x86 accepted?